spec/javascript/controllers/pinterest_controller_spec.js

```diff
@@ -0,0 +1,67 @@
+import { Application } from 'stimulus' ;
+import PinterestController from 'pinterest_controller' ;
+
+describe('PinterestController', () => {
```

Comment:
These are fairly repetitive, you could maybe include the core tests in another module and require them in each controller spec to DRY it up (passing in the params for that particular controller)